privacyidea/static/components/config/views/config.events.details.html

```diff
@@ -14,7 +14,7 @@
         Enable
     </button>
     <button class="btn btn-danger"
-        ng-click="delEvent(eventid)">
+        ng-click="confirmDeleteEvent(eventid)">
```

Comment:
This would be called 

~~~~html
ng-click="confirmDelete(delEvent, eventid, "Do you want to get rid of this annoying event?")
~~~~

Please then also check the translation of the question.

Reply:
fixed in 9c208ab4212d0e7fd5a0133faf6722115b3a40c9.